### PR TITLE
[spec] update scrollTop/Left pageX/Y definitions and make scroll even…

### PR DESCRIPTION
**Pull request by ymalik**
…t firing more clear

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,7 +179,19 @@
           <a data-link-type="dfn" href="https://drafts.csswg.org/cssom-view/#scrolling-events">section 12.2</a>
           of the CSSOM View module.
         </p>
+        <p>
+          This event should also be fired if a resize of the visual viewport causes either the
```
Comment (bokand) [inline index.html]: s/should/must

File: index.html [hunk 2]
```diff
@@ -118,20 +118,20 @@
...
           <dt><dfn>scrollLeft</dfn></dt>
           <dd>
-            Returns the distance from the left edge of the visual viewport to the left edge of the layout viewport.
+            Returns the location of the left edge of the visual viewport relative to the left edge of the layout viewport in CSS pixels.
           </dd>
 
           <dt><dfn>pageX</dfn></dt>
           <dd>
-            Returns the location of the left edge of the visual viewport from the left edge of the document.
+            Returns the x-coordinate, relative to the <a href="https://drafts.csswg.org/css-display-3/#initial-containing-block">initial containing block</a> origin, of the left of the visual viewport in CSS pixels.
```
Comment (bokand) [inline index.html]: "of the left of the visual viewport" sounds odd. Maybe "of the left edge of the visual viewport"?

File: index.html (hunk 1 above)
Comment (ymalik) [inline index.html]: Done!

File: index.html (hunk 2 above)
Comment (ymalik) [inline index.html]: Done!

**Pull request merged**